src/actions/LoginActions.tsx

```diff
@@ -249,13 +249,22 @@ export function initializeAccount(navigation: NavigationBase, account: EdgeAccou
       showError(error)
     }
 
+    // Post login stuff:
     if (!newAccount && !hideSurvey && !getDeviceSettings().isSurveyDiscoverShown && config.disableSurveyModal !== true) {
       // Show the survey modal once per app install, only if this isn't the
       // first login of a newly created account and the user didn't get any
       // other modals or scene changes immediately after login.
       await Airship.show(bridge => <SurveyModal bridge={bridge} />)
       await writeIsSurveyDiscoverShown(true)
     }
+
+    if ((await getCountryCodeByIp()) === 'GB') {
+      await showPersistentWarning(lstrings.warning_uk_risk, {
+        onPress: async () => {
+          await openLink('https://edge.app/due-diligence/')
+        }
+      })
```

Comment:
Yeah, this would be better as:

```ts
await Airship.show(bridge => <AlertDropdown
   bridge={bridge}
   message={lstrings.warning_uk_risk}
   persistent
   warning
   onPress={async () => {
       await openLink('https://edge.app/due-diligence/')
     }}
   />)
```